core/store/src/metrics.rs

```diff
@@ -326,3 +326,20 @@ pub mod flat_state_metrics {
         .unwrap()
     });
 }
+pub static COLD_MIGRATION_INITIAL_WRITES: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
nit: COLD_STORE_MIGRATION_BATCH_WRITE_COUNT
- migration and initial seem a bit redundant and other cold store metrics have COLD_STORE_ prefix 
- it would be nice to show that this is about a batch and not a single key value